include/openenclave/bits/evidence.h

```diff
@@ -62,29 +56,88 @@ typedef struct _oe_claim
     size_t value_size;
 } oe_claim_t;
 
-/**
+/*
  * Claims that are known to OE that every attestation plugin should output.
  */
+
+/**
+ * Claims version.
+ */
 #define OE_CLAIM_ID_VERSION "id_version"
+
+/**
+ * Security version of the enclave. (SVN for SGX).
```

Comment:
nit: remove duplicate period
```suggestion
 * Security version of the enclave (SVN for SGX).
```